src/taoensso/sente.cljc

```diff
@@ -55,6 +55,7 @@
     :last-close     - ?{:udt _ :reason _}, with reason e/o
                         #{nil :requested-disconnect :requested-reconnect
                          :downgrading-ws-to-ajax :unexpected}
+    :next-reconnect - ?udt of next reconnect attempt
```

Comment:
Would prefer we name this `:udt-next-reconnect` and describe it as "Approximate udt of next scheduled auto-reconnect attempt".